sydent/config/__init__.py

```diff
@@ -72,6 +73,12 @@
         # This whitelist overrides `ip.blacklist` and defaults to an empty
         # list.
         "ip.whitelist": "",
+        # A list of homeservers that are allowed to register with this identity server. Defaults to
+        # allowing all homeservers. If a list is specified, the config option `disable_v1_access` must be
+        # set to 'true'.
+        "homeserver_allow_list": "",
+        # If set to 'true`, entirely disable access via the V1 api.
+        "disable_v1_access": "false",
```

Comment:
I think we've decided to always use `enable_*` rather than `disable_*` for consistency